### ISSUE TITLE
Visually out of place drop down in pull requests views with Safari 

### ISSUE THREAD

**Issue by mjbvz** [microsoft/vscode-pull-request-github#2426]
**Repro**
1. Using the latest nightly in a codespace loaded in Safari
2. Create a pull request

**Bug**
In the pull requests view, the drop downs are using the standard aqua-styling from safari:

<img width="404" alt="Screen Shot 2021-01-27 at 2 32 01 PM" src="https://user-images.githubusercontent.com/12821956/106063150-9c6d9200-60ac-11eb-9729-b1905eaf4725.png">

This looks out of place compared to our other dropdowns:

<img width="240" alt="Screen Shot 2021-01-27 at 2 32 33 PM" src="https://user-images.githubusercontent.com/12821956/106063209-ac857180-60ac-11eb-8fb3-7c9e1e5447db.png">

**Comment by alexr00** [comment]
@burkeholland in case you want to work on this!

**Comment by Yash621** [comment]
@mjbvz can I work on this ?

**Comment by alexr00** [comment]
@Yash621 you are welcome to work on this!

**Comment by Yash621** [comment]
@alexr00 can you guide me on this ,like how should I go about it ?

**Comment by alexr00** [comment]
The dropdowns are here:

https://github.com/microsoft/vscode-pull-request-github/blob/b4bb8a6c63109e838ae54797ce608c61123f19b3/webviews/createPullRequestView/app.tsx#L12-L49

I haven't done any investigation into this issue, but I assume there is some styling that can be done to make the dropdowns look more like VS Code dropdowns. Styles are here: https://github.com/microsoft/vscode-pull-request-github/blob/09a489f8dd15d0a6c98e927879e863d234ebd68a/webviews/createPullRequestView/index.css

**Comment by alexr00** [comment]
Looks like we lost some of the original styling (the arrow on the right of the select is gone):
![image](https://user-images.githubusercontent.com/38270282/148943252-a6d96546-5d68-43d4-8e60-9938db3b3f13.png)

The select now also has rounded corners, which isn't consistent with other selects in VS Code:
![image](https://user-images.githubusercontent.com/38270282/148943369-c0d95544-ae81-4cd1-834b-9466bca0a957.png)

The colors look good though. Do you know if there's a way to fix the lost styling?

**Comment by saifahn** [comment]
@alexr00 

Hey there!

Is there a way to install the version of the extension that I'm working on in Codespaces? I had a look through [this page on remote development and Codespaces](https://code.visualstudio.com/api/advanced-topics/remote-extensions), but couldn't get it to work.

I tried opening my repository in Codespaces, then packaging the extension into a VSIX using `vsce package` and installing that way, but I got an error:

```
Command 'Extensions: Install from VSIX...' resulted in an error (Unable to install extension 'github.vscode-pull-request-github' as it is not compatible with VS Code '1.69.2'.)
```

Also, it seems like this GitHub Pull Requests and Issues extension is installed by default in Codespaces and can't be removed or have other versions installed 🤔 

I'm thinking I might need to use Insiders or something, but I don't think I can use that with Safari, right?

Any pointers would be great 😄
